guideline.md

```diff
@@ -0,0 +1,159 @@
+
+## PyLadies Germany Spokesperson
+
+PyLadies Germany will have a spokesperson at the PySV to mediate between the two groups and inform other chapters (existing and new) about the process. This person is responsible for supporting chapters to follow these guidelines and tracked expenses are reported. They are not responsible to singularly deliver these results as this is a shared responsibility of all chapters. If they feel that the process is being misused or have any other concerns, they should raise it with the rest of the approval group for discussion and the PySV.
+
+The role should rotate and is open to any PyLady in Germany. A mechanism for this rotation and the period of tenure is not yet defined. 
+
+As this person will be from one of the chapters represented in the approval group, they should refrain from being the second approver if the other representative of their chapter has already approved.
+PyLadies Germany Spokesperson attendance of PySV meetings:
+
+The spokesperson's involvement will also include attending PySV meetings. The frequency and schedule of these meetings will be determined jointly by PyLadies and PySV.
+
+## Reporting:
+
+The platform for approving and tracking requests should be as publicly open as possible so that any PyLady in Germany has access. Therefore no individual's personal information should be shared unless it is relevant (individuals can be referred to by their role i.e. event speaker). Invoices should not be posted there.
+
+Tickets will be labeled by chapter and year and the platform should allow PyLadies to see at a glance what has been requested, approved, and reimbursed. Anyone should be able to make a request and there should be guidelines on how to do so.
+
+A general update on notable events and activities should be sent to PySV every six months. This helps PySV understand the needs and accomplishments of PyLadies. The report should be sent by the spokesperson with the support of the other approval group members.
+
+Details of all expenditures must be reported so that accounts can be properly reconciled. These are sent directly to the PySV with the [pyladies@python-verband.org](mailto:pyladies@python-verband.org) email in CC by the fund spokesperson in regular intervals (TBD: which interval, likely 6 months or yearly).
+
+## Current Approvals Group
+
+Email: germany@pyladies.com (this is a Google group)
+
+*Access to the email is only granted to current approval group members and should have 2FA enabled.*
+
+Approval Platform: GitHub Project (this is public and should contain no personal information)
+
+**Spokesperson:** [Jessica Greene](https://github.com/sleepypioneer/) [Berlin]
+XXXX XXXX [Berlin]
+XXXX XXXX [Hamburg]
```

Comment:
@terezaif are you happy to be added here for now? We can also leave Hamburg blank for the time being